articles/communication-services/concepts/advanced-messaging/whatsapp/pricing.md

```diff
@@ -38,7 +38,7 @@ When you connect your WhatsApp Business account to Azure, Azure Communication Se
 - WhatsApp will charge [per-message for template messages](https://developers.facebook.com/docs/whatsapp/pricing/updates-to-pricing#per-message-pricing) instead of per-conversation.
 - Utility template messages sent within an open customer service window [will become free](https://developers.facebook.com/docs/whatsapp/pricing/updates-to-pricing#free-utility-templates-in-the-customer-service-window).
 
-See [Pricing Updates on the WhatsApp Business Platform](https://developers.facebook.com/docs/whatsapp/pricing/updates-to-pricing/) for additional details.
+See [Pricing Updates on the WhatsApp Business Platform](https://developers.facebook.com/docs/whatsapp/pricing/updates-to-pricing/) for additional details. To view the original amount billed by Meta, prior to any Azure benefits or adjustments, refer to the Quantity column in your [Azure usage details file](https://learn.microsoft.com/en-us/azure/cost-management-billing/understand/download-azure-daily-usage).
```

Comment:
```suggestion
See [Pricing Updates on the WhatsApp Business Platform](https://developers.facebook.com/docs/whatsapp/pricing/updates-to-pricing/) for additional details. To view the original amount billed by Meta, prior to any Azure benefits or adjustments, refer to the Quantity column in your [Azure usage details file](https://learn.microsoft.com/azure/cost-management-billing/understand/download-azure-daily-usage).
```
this will resolve the build warning